### ISSUE TITLE
albacore > 1.2.4 

### ISSUE THREAD

**Issue by haansi** [mitenjain/signalAlign#3]
hi guys,

Thanks for your work! Are you planning to supported albacore > 1.2.4 fast5 files anytime soon?
Here's a sample entry from run.err:
`[NanoporeRead:_initialize]ERROR alba_read_1_ch_1_strand.fast5 unsupported version 1.2.5 `

**Comment by mitenjain** [comment]
Hi there, 

Sorry for the delayed reply. A new, revamped version is coming real soon. I'll post when it arrives.

**Comment by nmb85** [comment]
Any updates about the new version? Keen to give it a shot!

**Comment by mitenjain** [comment]
Yes. We have a new version that should become available in a few days. I will put the link here then. Thank you.

**Comment by nmb85** [comment]
Thank you!

**Comment by nmb85** [comment]
Hi @mitenjain, it looks like albacore has just been deprecated and guppy is the replacement. Will this set back signalAlign development?

**Comment by mitenjain** [comment]
It will require some minor refactoring but shouldn't take too long to add. We are working on that right now as well.

**Comment by nmb85** [comment]
Great! Thanks for the update!
